.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,31 +1,26 @@
-##### SUMMARY
-<!--- Describe the change, including rationale and design decisions -->
+<!--- Describe the change below, including rationale and design decisions -->
 
-<!--- If you are fixing an existing issue, please include "Fixes #nnn" in your
-commit message and your description; but you should still explain what
-the change does.-->
+<!--- HINT: Include "Fixes #nnn" if you are fixing an existing issue -->
 
-##### ISSUE TYPE
-<!--- Pick one below and delete the rest: -->
- - Feature Pull Request
- - New Module Pull Request
- - Bugfix Pull Request
- - Docs Pull Request
+### Type of pull request
+<!--- Pick one below and delete the rest -->
+- New module
```

Comment:
If we're re-ordering the list of PR types, I'd vote for alphabetized:
Bugfix
Docs
Feature
New module

Reply:
Right, I ordered them by what new contributors would most likely need. But I confess that even that is arbitrary ;-)